openupgrade_scripts/scripts/project/15.0.1.2/post-migration.py

```diff
@@ -0,0 +1,108 @@
+from openupgradelib import openupgrade
+
+
+def _convert_field_m2o_to_m2m(env):
+    # Convert m2o to m2m in 'project.task'
+    openupgrade.m2o_to_x2m(
+        env.cr,
+        env["project.task"],
+        "project_task",
+        "user_ids",
+        openupgrade.get_legacy_name("user_id"),
+    )
+
+
+def _create_missing_project_task_stage_personal(env):
+    tasks = env["project.task"].with_context(active_test=False).search([])
+    tasks._populate_missing_personal_stages()
```

Comment:
This is doing nothing, as there aren't any `project.task.stage.personal` record.

Reply:
Yes, there are records. The table of model project.task.stage.personal is project_task_user_rel, which is also the m2m table between project.task and res.users, which already exists in v14.
![Selection_502](https://user-images.githubusercontent.com/25005517/182344195-51b245a1-26d0-4076-892a-0d5771462ae0.png)

Reply:
But such table is useless now AFAIK. Which are their contents in v14?

Reply:
Sorry, I was wrong, confused by other thing. This table is new in v15, and is the transformation of the m2o field `user_id` to `user_ids`, which are the assignees to the task. Now each task can have more that one assignee.